### PR TITLE
sci-astronomy/stellarium: Version bump to 0.15.1

### PR DESCRIPTION
**Pull request by g-braeunlich**
Package-Manager: portage-2.3.0

### CODE REVIEW

File: sci-astronomy/stellarium/stellarium-0.15.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	zh-CN zh-HK zh-TW zu
+	)
+
+for X in "${LANGS[@]}" ; do
+	IUSE+=" l10n_${X}"
+done
+unset X
+
+src_prepare() {
+	default
```
Comment (SoapGentoo) [inline sci-astronomy/stellarium/stellarium-0.15.1.ebuild]: always use `cmake-utils_src_prepare` with cmake, never use `default` in such cases

File: sci-astronomy/stellarium/stellarium-0.15.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+		CMakeLists.txt || die
+	use debug || append-cppflags -DQT_NO_DEBUG #415769
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DENABLE_NLS="$(usex nls)"
+		-DENABLE_MEDIA="$(usex media)"
+	)
+	CMAKE_IN_SOURCE_BUILD=1 cmake-utils_src_configure
```
Comment (SoapGentoo) [inline sci-astronomy/stellarium/stellarium-0.15.1.ebuild]: define this globally and not just before `cmake-utils_src_configure`
Optimally, making the build system work out-of-source should always be our goal (and then submit patches), but don't feel obliged

File: sci-astronomy/stellarium/stellarium-0.15.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+	# use the more up-to-date system fonts
+	rm "${ED%/}"/usr/share/stellarium/data/DejaVuSans{Mono,}.ttf || die
+	dosym ../../fonts/dejavu/DejaVuSans.ttf /usr/share/stellarium/data/DejaVuSans.ttf
+	dosym ../../fonts/dejavu/DejaVuSansMono.ttf /usr/share/stellarium/data/DejaVuSansMono.ttf
+
+	if use stars ; then
+		insinto /usr/share/${PN}/stars/default
+		doins "${DISTDIR}"/stars_[45678]_[12]v0_1*.cat
+	fi
+	newicon doc/images/stellarium-logo.png ${PN}.png
```
Comment (SoapGentoo) [inline sci-astronomy/stellarium/stellarium-0.15.1.ebuild]: requires `eutils`

File: sci-astronomy/stellarium/stellarium-0.15.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+		mirror://sourceforge/stellarium/stars_5_2v0_1.cat
+		mirror://sourceforge/stellarium/stars_6_2v0_1.cat
+		mirror://sourceforge/stellarium/stars_7_2v0_1.cat
+		mirror://sourceforge/stellarium/stars_8_2v0_1.cat
+	)"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="debug nls media stars"
```
Comment (SoapGentoo) [inline sci-astronomy/stellarium/stellarium-0.15.1.ebuild]: sort alphabetically

File: sci-astronomy/stellarium/stellarium-0.15.1.ebuild (hunk 2 above)
Comment (g-braeunlich) [inline sci-astronomy/stellarium/stellarium-0.15.1.ebuild]: Seems to work out of the box without CMAKE_IN_SOURCE_BUILD=1

**Pull request closed without merging**